### PR TITLE
Gives you the option to emergency eject clones

### PR DESCRIPTION
**Pull request by DGamerL**
## What Does This PR Do
Adds an alert whether you want to change the access restrictions or emergency eject a clone from the cloning pod

## Why It's Good For The Game
Grrr removed functionality that was very useful

## Images of changes
image

## Testing
See above, ejected a clone

## Changelog
:cl:
fix: Added the option back to emergency eject people from the cloning pod
/:cl:

### CODE REVIEW

**Review by S34NW — COMMENTED** [review]
Bro that TGUI is scuffed to hell, any chance you can fix it?

File: code/game/machinery/clonepod.dm [hunk 1]
```diff
@@ -563,11 +563,17 @@
...
-		if(allowed(user))
-			locked = !locked
-			to_chat(user, "<span class='notice'>Access restriction is now [locked ? "enabled" : "disabled"].</span>")
-		else
+		if(!allowed(user))
 			to_chat(user, "<span class='warning'>Access denied.</span>")
+			return
+
+		switch(tgui_alert(user, "Do you want to change access restrictions or perform an emergency ejection of [src]?", "Cloning pod", list("Change access restrictions", "Emergency ejection")))
+			if("Change access restrictions")
```
Comment (GDNgit) [inline code/game/machinery/clonepod.dm:571]:
```suggestion
			if("Change access")
```
Magic

**Review by Burzah — APPROVED** [review]

File: code/game/machinery/clonepod.dm [hunk 2]
```diff
@@ -563,11 +563,17 @@
...
 			to_chat(user, "<span class='warning'>Access denied.</span>")
+			return
+
+		switch(tgui_alert(user, "Change access restrictions or perform an emergency ejection of [src]?", "Cloning pod", list("Change access", "Emergency ejection")))
+			if("Change access")
+				locked = !locked
+				to_chat(user, "<span class='notice'>Access restriction is now [locked ? "enabled" : "disabled"].</span>")
+			if("Emergency ejection")
+				eject_clone(TRUE) // GET OUT
+				to_chat(user, "<span class='warning'>You force [src] to eject it's clone!</span>")
```
Comment (lewcc) [inline code/game/machinery/clonepod.dm:576]:
```suggestion
				to_chat(user, "<span class='warning'>You force [src] to eject its clone!</span>")
```

**Review by lewcc — COMMENTED** [review]
go go grammer team

**Pull request merged**